web/autograder_service.go

```diff
@@ -581,13 +581,31 @@ func (s *AutograderService) RebuildSubmission(ctx context.Context, in *pb.Rebuil
 		s.logger.Errorf("ApproveSubmission failed: submitter has no access to the course")
 		return nil, status.Error(codes.PermissionDenied, "submitter has no course access")
 	}
-	submission, err := s.rebuildSubmission(ctx, in)
+	submission, err := s.rebuildSubmission(in)
 	if err != nil {
 		return nil, err
 	}
 	return submission, nil
 }
 
+// RebuildSubmissions runs tests for all submissions for the given assignment ID.
```

Comment:
Add access policy to the doc. I guess only teachers and admin are allowed to call this.

Reply:
Fixed.